lighthouse-core/config/config.js

```diff
@@ -664,22 +664,6 @@ class Config {
     return {categories, requestedAuditNames: includedAudits};
   }
 
-  /**
-   * @param {LH.Config.Json} config
-   * @return {Array<{id: string, title: string}>}
-   */
-  static getCategories(config) {
```

Comment:
this is only used in two tests now (maybe after the extension was removed?), so it seemed easier just to delete than to deal with localization